index.js

```diff
@@ -0,0 +1,194 @@
+import {
+  startGame,
+  gameBord,
+  replayGame,
+  allcells,
+  nextPlayer,
+  playersForm,
+  playerOneName,
+  playerTwoName,
+  alertMsg,
+  gameOver,
+  // eslint-disable-next-line import/extensions
+} from './dom.js';
+
+// eslint-disable-next-line func-names
+const domElements = (function () {
+  const inAlertMsg = (msg) => {
+    alertMsg.innerHTML = msg;
```

Comment:
- [ ] This still counts as DOM manipulation. Please move this to dom.js, along with other similar code lines.